omnigibson/object_states/factory.py

```diff
@@ -55,13 +16,15 @@
     "freezable": [Frozen],
     "heatable": [Heated],
     "heatSource": [HeatSourceOrSink],
+    "meltable": [],
+    "mixing_wand": [],
```

Comment:
nit: maybe we should find a more formal name for this?

Reply:
change to `mixing_tool`